members.yaml

```diff
@@ -304,3 +403,7 @@ components:
         title:
           type: string
           enum: [VORSTANDSVORSITZENDER, SYSTEMADMINISTRATOR, KASSIERER, FLUGWART, IMBETRIEBSKONTROLLTURMARBEITEND]
+    Password:
+      type: string
+      format: password
+      description: "Mind: 8 Zeichen, 1 Zahl, 1 Buchstabe"
```

Comment:
Basically im fine with this requirements but where do they come from? PO requirement or decision by yourself?

Reply:
is vom PO / Kunden, ich denk mir so einen Schwachsinn doch nicht aus :rofl:

Reply:
hehe. Ok dann hab ich das entweder verschlafen oder es ist im frontend untergegangen.